app/modules/encounters/resources.py

```diff
@@ -81,11 +82,12 @@ def post(self, args):
         except Exception:
             pass
 
-        result_data, message, error = current_app.edm.request_passthrough_result(
-            'encounter.data', 'post', {'data': data}, ''
-        )
-        if not result_data:
-            abort(success=False, passed_message=message, message='Error', code=400)
+        try:
+            result_data = current_app.edm.request_passthrough_result(
+                'encounter.data', 'post', {'data': data}, ''
+            )
+        except HoustonException as ex:
+            abort(400, 'Error', success=False, passed_message=ex.message)
```

Comment:
400 only as that's what the existing code did. Is there any reason this did not use the edm status code (other than it wasn't actually available here in the existing code?

Reply:
this ends up being kinda moot point i guess as i completely gut this `post()` part out of here via PR177!